src/main/frontend/ui.cljs

```diff
@@ -715,6 +715,9 @@
   < {:did-catch
      (fn [state error _info]
        (log/error :exception error)
+       (notification-handler/show!
```

Comment:
I think this is a good idea for the default error screen but I don't this is a good idea for all 10 uses of this fn. For example, this fn is called when a query has invalid syntax. This results in this notification popping up each time the query is rendered or when the cursor moves to it. This would be quite annoying for users - https://www.loom.com/share/2ad965ff7916451ab74c3aaae416dd98 . Maybe make this notification toggled through an option or create a new helper `catch-error-and-notify` and use it for default error screen

Reply:
I feel introducing an option is a bit over-engineering. Just copy-paste the function as `catch-error-and-notify` 🤣